conf/defaults.ini

```diff
@@ -176,6 +176,9 @@ cookie_secure = false
 # set cookie SameSite attribute. defaults to `lax`. can be set to "lax", "strict" and "none"
 cookie_samesite = lax
 
+# set to true if you want to allow browsers to render Grafana in a <frame>, <iframe>, <embed> or <object>. default is false.
+allow_embedding = false
```

Comment:
:+1: We should default the secure option as long as it's still easy to get started on a local dev machine.

Reply:
@bergquist What do you mean, you want this to default to true if in dev environment?

Reply:
No. I just wanted to mention that I think you made the right decision to enable these headers by default :)

Reply:
Great :+1: